core/js/typespecs/app_types.ts

```diff
@@ -786,7 +799,7 @@ const patchProjectScoreSchema = z
     name: projectScoreSchema.shape.name.nullish(),
     description: projectScoreSchema.shape.description,
     score_type: projectScoreSchema.shape.score_type.nullish(),
-    categories: projectScoreSchema.shape.categories,
+    config: projectScoreSchema.shape.categories,
```

Comment:
why?

Reply:
This shouldn't be changed. I briefly flirted iwth renaming the field before realizing how big of an undertaking that would be.